FutureFeatures.md

```diff
@@ -273,14 +273,19 @@ quadruple precision.
 
 ## Floating point library intrinsics
 
-These operations aren't needed because they can be implemented in WebAssembly
-code and linked into WebAssembly modules as at small size cost, and this avoids
-a non-trivial specification burden of their semantics/precision. Adding these
-intrinsics would allow for better high-level backend optimization of these
-intrinsics that require builtin knowledge of their semantics. On the other
-hand, a code generator may continue to statically link in its own
-implementation since this provides greater control over precision/performance
-tradeoffs.
+These operations aren't needed for the MVP because they can be implemented in
+WebAssembly code and linked into WebAssembly modules at small size cost (as is
+traditionally done through C libraries such as libm). This avoids a non-trivial
+specification burden for their semantics, precision, and performance.
```

Comment:
Yes indeed, let's talk about the transcendental math caches, a topic many of us are familiar with. This is an example where some implementations were faster than others on some workloads, and it isn't a simple matter of the others "lagging behind" and needing to "work harder". And while Sunspider may have started it, it turns out that some real-world applications benefit from the caches as well. So I'll add another bullet point to my list:
 - there's no temptation for implementations to optimize for some input patterns at the expense of others.

(It's true that we face this temptation in other areas, but that doesn't mean we shouldn't wish to avoid it here.)